templates/tease-search.twig

```diff
@@ -46,7 +46,7 @@
 						<a href="{{ post.link() }}" class="search-result-item-headline">{{ post.title|e('wp_kses_post')|raw }}</a>
 					{% endif %}
 
-					{% if ( not is_action ) %}
+					{% if ( not is_action and 'default' != post._wp_page_template ) %}
```

Comment:
shouldn't we also add 'evergreen' here?

Reply:
So, since we do not show dates for Action pages already, does that mean that we do not want visible dates for any Page result?
Can we then simplify the whole `if` statement by checking that it is not a page post_type?
@dpivo @comzeradd

Reply:
Yes, good point. I guess if we block all these page types there is no page actually left, only posts.

Reply:
Yes, I was thinking the same.
We could hide dates for all WordPress 'pages'.
This could help to check against the post type
https://timber.github.io/docs/reference/timber-post/#type

Reply:
Noted, I'll add the changes